<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ChatMainWindow</class>
 <widget class="QMainWindow" name="ChatMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qt D-Bus Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QTextBrowser" name="chatHistory" >
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>Messages sent and received from other users</string>
        </property>
        <property name="acceptRichText" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Message:</string>
          </property>
          <property name="buddy" >
           <cstring>messageLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="messageLineEdit" />
        </item>
        <item>
         <widget class="QPushButton" name="sendButton" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>1</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Sends a message to other people</string>
          </property>
          <property name="whatsThis" >
           <string/>
          </property>
          <property name="text" >
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuit" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionChangeNickname" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuQuit" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="text" >
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionChangeNickname" >
   <property name="text" >
    <string>Change nickname...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>chatHistory</tabstop>
  <tabstop>messageLineEdit</tabstop>
  <tabstop>sendButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>messageLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sendButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>299</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel" >
     <x>744</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered(bool)</signal>
   <receiver>ChatMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
